### PR TITLE
iOS recycler cells implementation

### PR DESCRIPTION
**Pull request by SergioEstevao**
This PR implement the display of the React cells inside the table view.

It also implements all the actions of the cells and toolbars associated.

@maxme do you mind give it a spin?

### CODE REVIEW

**Comment by maxme** [comment]
I got a JS error when trying to move a cell down:

https://bia.is/s/YGif/recycler-view-ios.mov

This happens everytime right after loading the project:

* press the "down" button on any of the cells -> 💥

File: example/App.ios.js [hunk 1]
```diff
@@ -12,7 +12,9 @@ import {
   View,
   Image,
   Button,
-  TouchableNativeFeedback
+  TouchableNativeFeedback,
```
Comment (maxme) [inline example/App.ios.js:15]: This import is unused (`TouchableNativeFeedback` not supported in iOS)

Comment (maxme) [reply example/App.ios.js:15]: (Makes me think that we should add the same eslint config that we have in `gutenberg-mobile`)

**Comment by maxme** [comment]
I did a quick and dirty patch to factorize the Android and iOS example, looks good on both platforms and the changes are minimal:

See the branch: `issue/ios_recycler_cells_implementation-common-example` or the commit https://github.com/godness84/react-native-recyclerview-list/commit/6ce419fc16940e1cc8b4e9730547bc77cf3ef268

**Comment by maxme** [comment]
@SergioEstevao good job with this! PR looks good. I'm actually quite surprised that visually it looks very similar (animations and transitions) on both platforms!

:shipit:

**Comment by seojeongah80** [comment]
)

**Pull request merged**